lib/hashicorptools/code_deploy.rb

```diff
@@ -33,36 +70,22 @@ def deploy
                  g.log.first
                end
 
-
-      puts "deploying commit: #{commit.sha} #{commit.message}"
-
-      create_deployment(commit.sha, commit.message)
+      aws_regions.each do |aws_region|
```

Comment:
Should we use ruby threads here so this can happen in parallel?

Reply:
i considered this, but wasn't sure how the output should look since we are currently providing periodic debug output as the deploy progresses

Reply:
maybe we can prefix the debug output with the region name, e.g. `[us-east-1] something happened`?